### ISSUE TITLE
Please make the number of issues per page configurable

### ISSUE THREAD

**Issue by mapreri** [aplanas/kmanga#10]
I'd love to have different value of number of issues per page in the subscription page, as 25 is really too low for my testes.

It would be nice if it was configurable in the user preferences.

PS.  Thank you for the great service!!

**Comment by aplanas** [comment]
You can change that in the lower part of the page. There you can choose 25, 50, 100, 200 or 400 issues per page.

**Comment by aplanas** [comment]
Oh. Your idea is to add it into the user preferences. This makes sense

**Comment by mapreri** [comment]
Yep, sorry I didn't make it clear enough!  (the idea is that clicking that selector ~every time I get to the page is boring)
